src/libpmemfile-posix/dir.c

```diff
@@ -357,10 +357,9 @@ vinode_update_parent(PMEMfilepool *pfp,
  * Note: caller must hold WRITE lock on parent.
  * Must be called in a transaction.
  */
-struct pmemfile_vinode *
+TOID(struct pmemfile_inode)
 vinode_new_dir(PMEMfilepool *pfp, struct pmemfile_vinode *parent,
```

Comment:
It operates on vinode, so I'd say no. If you have better idea how naming should look like I'm listening.